File: strpos/pos.go
package main

import (
	"fmt"
	"strings"
)

func strStr(haystack string, needle string) int {
	return strings.Index(haystack, needle)
}

func main() {
	fmt.Println(strStr("leetcode", "leeto"))
	fmt.Println(strStr("sadbutsad", "sad"))
}

File: strpos/pos.php
<?php

class Solution {
    /**
     * @param String $haystack
     * @param String $needle
     * @return Integer
     */
    function strStr($haystack, $needle) {
        $res = strpos($haystack, $needle);

        if ($res === false) {
            return -1;
        }

        return $res;
    }
}

$s = new Solution();

var_dump($s->strStr("leetcode", "leeto"));